conda/meta.yaml

```diff
@@ -1,24 +1,49 @@
-{% set data= load_setup_py_data() %}
+{% set data = load_setup_py_data() %}
+
 package:
   name: ts-salobj
-  version: {{ GIT_DESCRIBE_TAG }}
+  version: {{ data.get('version') }}
 
 source:
-  git_url: https://github.com/lsst-ts/ts_salobj
-  git_rev: {{ TS_SALOBJ_VERSION }}
+  path: ../
+#  git_url: https://github.com/lsst-ts/ts_salobj
+#  git_rev: {{ TS_SALOBJ_VERSION }}
 
 build:
-  script: python -m pip install --ignore-installed .
-  skip: True #[win]
+  script: python -m pip install --no-deps --ignore-installed .
+  script_env:
+    - PATH
+    - PYTHONPATH
+    - LD_LIBRARY_PATH
+    - OSPL_HOME
+    - PYTHON_BUILD_VERSION
+    - PYTHON_BUILD_LOCATION
+    - TS_CONFIG_OCS_DIR
 
 test:
   requires:
     - pytest
     - pytest-flake8
     - pytest-cov
     - asynctest
-#  commands:
-#    - py.test
+    - numpy
+    - astropy
+    - jsonschema
+    - pyyaml
+    - boto3
+    - moto
+    - ts-dds
+    - ts-idl
+  source_files:
+    - python
+    - bin
+    - tests
+    - schema
```

Comment:
Is it worth adding docs to this list?

Reply:
this list in particular is what should be added for conda build test phase... So I don't think adding docs would do anything.